pom.xml

```diff
@@ -153,6 +153,63 @@
                 <artifactId>jcl-over-slf4j</artifactId>
                 <version>1.5.6</version>
             </dependency>
+				  <dependency>
+		    <groupId>javax.xml.bind</groupId>
+		    <artifactId>jaxb-api</artifactId>
+		    <version>2.3.0</version>
+		</dependency>
+		<dependency>
+		    <groupId>com.sun.xml.bind</groupId>
+		    <artifactId>jaxb-core</artifactId>
+		    <version>2.3.0</version>
+		</dependency>
+		<dependency>
+		    <groupId>com.sun.xml.bind</groupId>
+		    <artifactId>jaxb-impl</artifactId>
+		    <version>2.4.0-b180830.0438</version>
+		</dependency>
+		<dependency>
+		    <groupId>javax.activation</groupId>
+		    <artifactId>activation</artifactId>
+		    <version>1.1.1</version>
+		</dependency>
+		<dependency>
+		    <groupId>com.sun.xml.ws</groupId>
+		    <artifactId>jaxws-ri</artifactId>
+		    <version>2.3.0</version>
+		    <type>pom</type>
+		</dependency>
+		<dependency>
+		    <groupId>com.sun.xml.bind</groupId>
+		    <artifactId>jaxb-xjc</artifactId>
+		    <version>2.4.0-b180830.0438</version>
+		</dependency>
+		<dependency>
+		    <groupId>com.sun.activation</groupId>
+		    <artifactId>javax.activation</artifactId>
+		    <version>1.2.0</version>
+		</dependency>
+		<dependency>
+		    <groupId>javax.transaction</groupId>
+		    <artifactId>javax.transaction-api</artifactId>
+		    <version>1.2</version>
+		</dependency>
+		<dependency>
+		    <groupId>javax.annotation</groupId>
```

Comment:
Why did you have to add all these dependencies? Any reason?

Reply:
The major difference while going from Java 8 to Java 11 is the dependencies that were removed, this is one of those.

Reply:
I don't understand your reply. You are adding more dependencies. Not removing any.

Besides, from your message, I understand you haven't managed to get Bindaas running. Please submit pull requests only after confirming it does not break any features.

Reply:
I was able to get bindaas running after making all the changes.

Reply:
The dependency was removed in Java 11. I added that dependency which is still available for Java 11. I'll create a PR in dev branch for you to check